scss/civicrm/dashboard/_dashboard.scss

```diff
@@ -1,5 +1,8 @@
 $dashboard-dashlets-margin: 10px;
 
+@import 'scss/civicrm/dashboard/dashlets/getting-started';
```

Comment:
Can we use `@import 'scss/civicrm/dashboard/dashlets/**';` instead?

Reply:
It seems globs only work on the root sass files. I tried to use them here, but they didn't work :( might need to investigate how we can use globs in partials. Will leave this for later.